sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -506,12 +506,10 @@ def test_user_agent_default(self, resource_group, location, storage_account, sto
 
         def callback(response):
             self.assertTrue('User-Agent' in response.http_request.headers)
-            self.assertEqual(
-                response.http_request.headers['User-Agent'],
-                "azsdk-python-storage-blob/{} Python/{} ({})".format(
+            assert ("azsdk-python-storage-blob/{} Python/{} ({})".format(
```

Comment:
Just test the presence of the moniker, not the entire logic of UA creation

Reply:
Shouldn't we validate the contents/formatting of the UA? This helps to meet our release checklist item for verifying the user agent....

Reply:
now that we are using the core's UA policy, I think we don't need to "re-test" the formatting again. Feel free to correct me though :)